en_pud-ud-test.conllu

```diff
@@ -14245,9 +14245,9 @@
 2	Macedonian	Macedonian	ADJ	JJ	Degree=Pos	3	amod	3:amod	Proper=True
 3	army	army	NOUN	NN	Number=Sing	4	nsubj	4:nsubj	_
 4	advanced	advance	VERB	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	0	root	0:root	_
-5	as	as	ADV	RB	_	9	case	9:case	_
-6	far	far	ADV	RB	Degree=Pos	5	fixed	5:fixed	_
-7	as	as	ADP	IN	_	5	fixed	5:fixed	_
+5	as	as	ADV	RB	_	6	advmod	6:advmod	_
+6	far	far	ADV	RB	Degree=Pos	4	advmod	4:advmod	_
+7	as	as	SCONJ	IN	_	9	mark	9:mark	_
 8	the	the	DET	DT	Definite=Def|PronType=Art	9	det	9:det	_
 9	pass	pass	NOUN	NN	Number=Sing	4	obl	4:obl:as_far_as|13:nsubj	_
```

Comment:
attaches to "far", `6:obl:as`

Reply:
Great, thanks.  Keep `13:nsubj` as an enhanced dependency?

Reply:
Yes

Reply:
head should be 6 not 7

Reply:
sorry again :/